packages/dpkg.rb

```diff
@@ -0,0 +1,38 @@
+require 'package'
+
+class Dpkg < Package
+  description 'dpkg, a medium-level package manager for Debian'
+  homepage 'https://anonscm.debian.org/git/dpkg/'
+  version '1.19.0.5'
+  source_url 'https://salsa.debian.org/dpkg-team/dpkg/-/archive/master/dpkg-master.tar.gz'
+  source_sha256 'a64b257d89d95f2e199fb39c6b37bf9de9e15e2d1e1898060fa9c9af3899330e'
+
+    depends_on 'perl' => :build
+    depends_on 'pkgconfig' => :build
+    depends_on 'automake' => :build
+    depends_on 'make' => :build
+    depends_on 'gcc' => :build
+    depends_on 'linuxheaders' => :build
+    depends_on 'flex' => :build
+    depends_on 'bison' => :build
+    depends_on 'icu4c' => :build
+    depends_on 'gettext' => :build
+    depends_on 'autoconf' => :build
+    depends_on 'libtool' => :build
```

Comment:
You won't need to include any of these dependencies since they're already included in Chromebrew "core".

Reply:
OK, I will give you `icu4c` as a necessary dependency but all others are not necessary.  Also, `gcc` has been replaced by `gcc7`.  See https://github.com/skycocker/chromebrew/blob/master/tools/core_packages.txt for reference.